File: september/09-09/09-09.py
def all_unique(s):

    is_unique = True

    
    for i in range(len(s)-2):
        
        cut = s[i+1 : len(s)]

        if s[i] in cut:
            is_unique = False

        
    print(is_unique)    
              
    return is_unique

all_unique("!@#*$%^&*()aA")

File: september/09-09/09-09.js
function allUnique(str) {
  
    let isUnique = true;

  for (let index = 0; index < str.length -2; index++) {
    
    let cut = str.substring(index +1, str.length);
    console.log(str[i])

    if(cut.includes(str[index])){
        isUnique = false;
    }
    
  }

  return isUnique;
}